docs/docs/Concepts/concepts-publish.md

```diff
@@ -1,81 +1,65 @@
 
 ## API access
 
-The **API access** pane presents code templates for integrating your flow into external applications.
-
-![](/img/api-pane.png)
+Langflow provides code snippets to help you get started with the Langflow API.
 
-<Tabs>
+To access the **API access** pane, click **Share**, and then click **API access**.
 
-<TabItem value="Python" label="Python">
+![API pane](/img/api-pane.png)
 
-The **Python** tab displays code to interact with your flow using the Python `requests` library.
+For more information, see [Run your flows from external applications](/get-started-quickstart#run-your-flows-from-external-applications).
 
-1. Copy and paste the code into a Python script.
-2. Run the script.
+### Input schema
 
-```python
-python3 python-test-script.py --message="tell me about something interesting"
-```
+Tweaks are added to the `payload` of requests to Langflow's `/run` endpoint to temporarily change component parameters within your flow.
+They don't modify the underlying flow configuration or persist between runs.
+To assist with formatting, you can define tweaks in Langflow's **Input Schema** pane before copying the code snippet.
 
-The response content depends on your flow. Make sure the endpoint returns a successful response.
+For more information, see [Use tweaks to apply temporary overrides to a flow run](/get-started-quickstart#use-tweaks-to-apply-temporary-overrides-to-a-flow-run).
 
-</TabItem>
+## Export
 
-<TabItem value="JavaScript" label="JavaScript" default>
+**Export** a flow to download it as a JSON file to your local machine.
 
-The **JavaScript API** tab displays code to interact with your flow in JavaScript.
+1. To **Export** your flow, in the **Playground**, click **Share**, and then click **Export**.
+2. To save your API keys with the flow, select **Save with my API keys**.
+Your flow is saved with any Global variables included.
 
-1. Copy and paste the code into a JavaScript file.
-2. Run the script.
+:::important
+If your key is saved as a Global variable, only the global variable you created to contain the value is saved. If your key value is manually entered into a component field, the actual key value is saved in the JSON file.
+:::
 
-```text
-node test-script.js "tell me about something interesting"
-```
+When you share your flow file with another user who has the same global variables populated, the flow runs without requiring keys to be added again.
 
-The response content depends on your flow. Make sure the endpoint returns a successful response.
+The `FLOW_NAME.json` file is downloaded to your local machine.
 
-</TabItem>
-<TabItem value="curl" label="curl" default>
+You can then **Import** the downloaded flow into another Langflow instance.
```

Comment:
You can remove this import/export section. I created a page dedicated to import/export in my upcoming PR. We can keep all import/export info there.